test/unit/metric_test.rb

```diff
@@ -72,6 +72,21 @@ def test_destroyable?
     assert metric_method.errors[:system_name].present?
   end
 
+  test 'fill owner' do
+    provider = FactoryBot.create(:simple_provider)
+
+    service = FactoryBot.create(:service, account: provider)
```

Comment:
The only difference between `:service` and `:simple_service` is that `:service` uses a `:provider_account` and `:simple_service` uses a `:simple_provider`, so a possible improvement would be removing the previous line (`provider = FactoryBot.create(:simple_provider)`), here using `service = FactoryBot.create(:simple_service)` and in line 84 doing `backend_api = BackendApi.create(name: 'API', system_name: 'api', account: service.provider)`

But this is completely optional 😄 And in fact I don't think you should repush if this is the only change bcz it is not important at all and we don't have much time 😄